pkg/cmd/server/start/master_args.go

```diff
@@ -362,23 +359,6 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 		config.ServiceAccountConfig.PublicKeyFiles = []string{}
 	}
 
-	// embed a default policy for generated config
```

Comment:
no objections from me. @smarterclayton, are we expecting this admission plugin to be enabled by default, and if so, does the default config for this plugin do what we want?

Reply:
> no objections from me. @smarterclayton, are we expecting this admission plugin to be enabled by default, and if so, does the default config for this plugin do what we want?

It does not.  No config means nothing.  This changes what an empty `openshift start master` does, but no one should be using that in production and production builds its config differently anyway.